django/test/utils.py

```diff
@@ -170,71 +171,140 @@ def teardown_test_environment():
     del mail.outbox
 
 
-def setup_databases(
```

Comment:
Small note that [this function is publicly documented so it cannot be removed](https://docs.djangoproject.com/en/5.1/topics/testing/advanced/#django.test.utils.setup_databases).

Reply:
Noted, thanks, I will push an update